src/btree/bt_handle.c

```diff
@@ -951,9 +952,10 @@ __btree_page_sizes(WT_SESSION_IMPL *session)
      */
     WT_RET(__wt_config_gets(session, cfg, "memory_page_max", &cval));
     btree->maxmempage = (uint64_t)cval.val;
+    min_pages_num = 10;
```

Comment:
For the sake of consistency with the rest of the code, I'd suggest using a macro here:
`#define WT_MIN_PAGES 10`

Reply:
got it